File: code/Java/Salario.java
package code.Java;

import java.util.Scanner;

public class Salario {
    private String nome;
    private double salario;

    public Salario() {}

    public void mensagem() {
        Scanner scan = new Scanner(System.in);
        System.out.println("Nome do salário:");
        this.nome = scan.nextLine();
        System.out.println("Salário:");
        this.salario = scan.nextDouble();

        System.out.printf("O funcionário %s tem o salário de R$%.2f no mes de abril\n", this.nome, this.salario);
        scan.close();
    }
}

File: code/Rust/salario.rs
fn main() {
    use std::io::stdin;
    let mut nome: String = String::new();
    let mut salario: String = String::new();
    println!("Nome do funcionário:");
    stdin().read_line(&mut nome).expect("Digite o Nome do funcionário!");
    println!("Salário:");
    stdin().read_line(&mut salario).expect("Digite o salário!");

    println!("O funcionário {}, tem um salário de R${} no mes de abril", nome, salario);
}